docs/source/models/gbd2017/causes/neonatal_other/index.rst

```diff
@@ -80,11 +80,15 @@ interventions with impacts on neonatal disorders, into a vivarium simulation.
 
 To model "other neonatal disorders" within the vivarium framework, we would need 
 to know:
-	- **prevalence**, in order to correctly initialize the right proportion of the population with other neonatal disorders;
-	- **birth prevalence**, in order to be able to correctly initialize new simulants throughout the simulation;
-	- **incidence**, in order for susceptible simulants to appropriately become prevalent cases;
-	- **remission**, in order for prevalent cases to appropriately remit;
-	- **excess mortality rate** (EMR), in order to appropriately calculate YLLs attributable to other neonatal disorers
+* **prevalence**, in order to correctly initialize the right proportion of the population with other neonatal disorders;
```

Comment:
You need a blank line before the start of this list:
```suggestion

* **prevalence**, in order to correctly initialize the right proportion of the population with other neonatal disorders;
```

Reply:
updated. thank you!